presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/RetryJdbcConfig.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc;
+
+import io.airlift.configuration.Config;
+
+import javax.validation.constraints.NotNull;
+
+import java.time.Duration;
+
+public class RetryJdbcConfig
+{
+    private java.time.Duration delay = Duration.ofSeconds(1);
+    private Duration maxDelay = Duration.ofMinutes(1);
+    private int maxRetries = 10;
+    private double delayFactor = 2.0;
+    private double jitterFactor = 0.2;
+
+    public Duration getDelay()
+    {
+        return delay;
+    }
+
+    @Config("retry-delay")
+    public RetryJdbcConfig setDelay(Duration delay)
+    {
+        this.delay = delay;
+        return this;
+    }
+
+    @NotNull
```

Comment:
It would be great if we could have some validation like Min value to be set

Reply:
Added Min and MinDuration annotations